diff -u cgoban-1.9.12.orig/configure cgoban-1.9.12/configure
--- cgoban-1.9.12.orig/configure	Thu Feb 28 04:33:00 2002
+++ cgoban-1.9.12/configure	Sun Sep 15 22:59:33 2002
@@ -386,9 +386,9 @@
     if test -n "`echo $ac_option| sed 's/[-a-z0-9.]//g'`"; then
       echo "configure: warning: $ac_option: invalid host type" 1>&2
     fi
-    if test "x$nonopt" != xNONE; then
-      { echo "configure: error: can only configure for one host and one target at a time" 1>&2; exit 1; }
-    fi
+ #   if test "x$nonopt" != xNONE; then
+  #    { echo "configure: error: can only configure for one host and one target at a time" 1>&2; exit 1; }
+   # fi
     nonopt="$ac_option"
     ;;
 
Common subdirectories: cgoban-1.9.12.orig/man6 and cgoban-1.9.12/man6
Common subdirectories: cgoban-1.9.12.orig/src and cgoban-1.9.12/src
Common subdirectories: cgoban-1.9.12.orig/wmslib and cgoban-1.9.12/wmslib
